### PR TITLE
ADD Changes to integrate with IOTAManager

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes errors found in the integration with the IoT Manager.

### CODE REVIEW

File: lib/templates/updateDevice.json [hunk 1]
```diff
@@ -96,6 +104,10 @@
           }
         }
       }
+    },
+    "static_attributes": {
+      "description": "Free form array of data to be appended to the target entity",
+      "type": "array"
     }
   }
 }
```
Comment (gtorodelvalle) [inline lib/templates/updateDevice.json]: Final blank line appreciated... ;)

**Comment by XavierVal** [comment]
LGTM

File: lib/services/devices/deviceService.js [hunk 2]
```diff
@@ -99,25 +100,34 @@ function createInitialEntityHandler(deviceData, newDevice, callback) {
...
+                        id: String(deviceData.name),
+                        attributes: []
+                    }
+                ],
+                updateAction: 'APPEND'
+            },
+            headers: {
+                'fiware-service': deviceData.service,
+                'fiware-servicepath': deviceData.subservice
+            }
```
Comment (gtorodelvalle) [inline lib/services/devices/deviceService.js]: I saw that in the initial entity creation request no `Fiware-Correlator` is included. Should it be included? Is it pending for a future PR? :) Thanks!

File: lib/services/devices/deviceService.js [hunk 3]
```diff
@@ -99,25 +100,34 @@ function createInitialEntityHandler(deviceData, newDevice, callback) {
...
-                {
-                    type: deviceData.type,
-                    isPattern: 'false',
-                    id: String(deviceData.name),
-                    attributes: []
-                }
-            ],
-            updateAction: 'APPEND'
+            url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
+                '/v1/updateContext',
```
Comment (gtorodelvalle) [inline lib/services/devices/deviceService.js]: The string `/v1/updateContext' is probably susceptible of being extracted to the `constants.js` file but I leave it totally to your discretion... ;)

Comment (dmoranj) [inline lib/services/devices/deviceService.js]: Usually I prefer not to extract this kind of paths to constant files, unless they are really mutable, as it reduces readability without really improving maintainability. I've moved other things, like the command and status prefixes, though, as they proved to be kind of unstable.

NTC

File: lib/services/devices/deviceService.js (hunk 2 above)
Comment (dmoranj) [inline lib/services/devices/deviceService.js]: Fixed in bff897c

File: lib/services/devices/deviceService.js (hunk 3 above)
Comment (gtorodelvalle) [inline lib/services/devices/deviceService.js]: NTC

**Comment by gtorodelvalle** [comment]
LGTM

**Pull request merged**